apis/python/tests/test_basic_anndata_io.py

```diff
@@ -417,3 +418,21 @@ def test_export_anndata(adata):
         assert readback.obsp[key].shape == adata.obsp[key].shape
     for key in adata.varp.keys():
         assert readback.varp[key].shape == adata.varp[key].shape
+
+
+def test_null_obs(adata):
+    tempdir = tempfile.TemporaryDirectory()
```

Comment:
While the pattern you're using here is the one from the existing tests, there are two different ways this could be slightly improved:

`tempfile.TemporaryDirectory()` works as a context manager, yielding the path of the temporary directory, so you can do:

```python
with tempfile.TemporaryDirectory() as output_path:
    ...  # do stuff with output_path
```

BUT in the specific case of a unit test, pytest offers a "fixture" which it magically provides based on parameter name (like `adata` here) called `tmp_path`, which will be a `pathlib.Path` that it automatically cleans up:

https://github.com/single-cell-data/TileDB-SOMA/blob/a78b1e6be9fe5c51d8fcfd317b1e05649424cd12/apis/python/tests/test_collection.py#L154

The other tests should probably be updated to match (but not right this minute).

Reply:
Good to know; I've replaced `tempdir` with the fixture for this particular test